shared/src/commonMain/kotlin/screens/login/LoginScreenUiState.kt

```diff
@@ -1,7 +1,7 @@
 package screens.login
 
 data class LoginScreenUiState(
-    var email: String = "yarin@gmail.com",
+    var email: String = "yarinn@gmail.com",
```

Comment:
Don't just put a default value like this, In Android we use if (Buildconfig.DEBUG)... but I don't know if that's possible in KMP. Just note that this should be cleaned for when the application will be distributed to our colleagues

Reply:
I don't seem to have any option to call BuildKonfig, but I will keep this in mind and do some research on it